src/Microsoft.Azure.WebJobs.Host/Loggers/FunctionInstanceLogger.cs

```diff
@@ -47,7 +48,12 @@ public Task LogFunctionCompletedAsync(FunctionCompletedMessage message, Cancella
         private void Log(LogLevel level, FunctionDescriptor descriptor, Guid functionId, string message, Exception exception = null)
         {
             ILogger logger = _loggerFactory?.CreateLogger(LogCategories.CreateFunctionCategory(descriptor.LogName));
-            logger?.Log(level, 0, message, exception, (s, e) => s);
+            var logData = new Dictionary<string, object>
+            {
+                ["Summary"] = message,
```

Comment:
Why put the message in the properties? I haven't seen us do this anywhere else in our logging code.